src/popover.ts

```diff
@@ -149,70 +163,132 @@ export function apply() {
     },
   });
 
-  const onClick = (event: Event) => {
-    const target = event.target;
-    if (!(target instanceof Element) || target?.shadowRoot) {
-      return;
-    }
-    const root = target.getRootNode();
-    if (!(root instanceof ShadowRoot || root instanceof Document)) {
+  const popoverTargetAttributesSupportedElementsSelector =
+    ':is(button, input[type="button"], input[type="submit"], input[type="image"], input[type="reset"])';
```

Comment:
Given this is a polyfill we might want to avoid selectors like `:is` which are fairly new. I don't think the `:is` adds anything here either.

```suggestion
    'button, input[type="button"], input[type="submit"], input[type="image"], input[type="reset"]';
```